common/lib/xmodule/xmodule/js/src/video/04_video_full_screen.js

```diff
@@ -11,82 +11,134 @@
             '</button>'
         ].join('');
 
-    // VideoControl() function - what this module "exports".
-        return function(state) {
-            var dfd = $.Deferred();
+    // The following properties and functions enable cross-browser use of the
+    // the Fullscreen Web API.
+    //
+    //     function getVendorPrefixed(property)
+    //     function getFullscreenElement()
+    //     function exitFullscreen()
+    //     function requestFullscreen(element, options)
+    //
+    //     For more information about the Fullscreen Web API see MDN:
+    //     https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API
+    //     way - you don't have to do repeated jQuery element selects.
+        var prefixedFullscreenProperties = (function() {
+            if ('fullscreenEnabled' in document) {
+                return {
+                    fullscreenElement: 'fullscreenElement',
+                    fullscreenEnabled: 'fullscreenEnabled',
+                    requestFullscreen: 'requestFullscreen',
+                    exitFullscreen: 'exitFullscreen',
+                    fullscreenchange: 'fullscreenchange',
+                    fullscreenerror: 'fullscreenerror'
+                };
+            }
+            if ('webkitFullscreenEnabled' in document) {
+                return {
+                    fullscreenElement: 'webkitFullscreenElement',
+                    fullscreenEnabled: 'webkitFullscreenEnabled',
+                    requestFullscreen: 'webkitRequestFullscreen',
+                    exitFullscreen: 'webkitExitFullscreen',
+                    fullscreenchange: 'webkitfullscreenchange',
+                    fullscreenerror: 'webkitfullscreenerror'
+                };
+            }
+            if ('mozFullScreenEnabled' in document) {
```

Comment:
Pointing out the capitalization difference here on "FullScreen" - assuming that's desired.

Reply:
Good catch

Reply:
It is. Mozilla seemed to have a different opinion on the camel-casing here.